pretty/public.go

```diff
@@ -134,19 +134,20 @@ func (cfg *Config) Fprint(w io.Writer, vals ...interface{}) (n int64, err error)
 }
 
 // Compare returns a string containing a line-by-line unified diff of the
-// values in got and want, using the CompareConfig.
+// values in a and b, using the CompareConfig.
 //
-// Each line in the output is prefixed with '+', '-', or ' ' to indicate if it
-// should be added to, removed from, or is correct for the "got" value with
-// respect to the "want" value.
-func Compare(got, want interface{}) string {
-	return CompareConfig.Compare(got, want)
+// Each line in the output is prefixed with '+', '-', or ' ' to indicate which
```

Comment:
Just say "marked with '+'" and such; keep the uses of the article a to a minimum where it doesn't mess things up grammatically ;).